seacrowd/sea_datasets/xquad/xquad.py

```diff
@@ -0,0 +1,151 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{Artetxe:etal:2019,
+      author    = {Mikel Artetxe and Sebastian Ruder and Dani Yogatama},
+      title     = {On the cross-lingual transferability of monolingual representations},
+      journal   = {CoRR},
+      volume    = {abs/1910.11856},
+      year      = {2019},
+      archivePrefix = {arXiv},
+      eprint    = {1910.11856}
+}
+"""
+
+_DATASETNAME = "xquad"
+
+_DESCRIPTION = """\
+XQuAD (Cross-lingual Question Answering Dataset) is a benchmark dataset for evaluating cross-lingual question answering performance.
+The dataset consists of a subset of 240 paragraphs and 1190 question-answer pairs from the development set of SQuAD v1.1 together (Rajpurkar et al., 2016)
+with their professional translations into ten languages in their original implementation: Spanish, German, Greek, Russian, Turkish, Arabic, Vietnamese, Thai, Chinese, and Hindi and two in this dataloader: Vietnamese & Thai
+"""
+
+_HOMEPAGE = "https://github.com/google-deepmind/xquad"
+
+_LICENSE = Licenses.CC_BY_SA_4_0.value
+
+_URLS = "https://raw.githubusercontent.com/google-deepmind/xquad/master/"
+
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class XQuADDataset(datasets.GeneratorBasedBuilder):
+    """
+    XQuAD (Cross-lingual Question Answering Dataset) is a benchmark dataset for evaluating cross-lingual question answering performance.
+    The dataset consists of a subset of 240 paragraphs and 1190 question-answer pairs from the development set of SQuAD v1.1 together
+    with their professional translations into ten languages: Spanish, German, Greek, Russian, Turkish, Arabic, Vietnamese, Thai, Chinese, and Hindi.
+    """
+
+    subsets = ["xquad", "xquad.vi", "xquad.th"]
```

Comment:
Hi, the `xquad` isn't necessary to be implemented, can you remove it?
```suggestion
    subsets = ["xquad.vi", "xquad.th"]
```

and for the sake on reproducibility in testing, can you check add add these two?
```bash
python -m tests.test_seacrowd seacrowd/sea_datasets/xquad/xquad.py --subset_id xquad.vi
python -m tests.test_seacrowd seacrowd/sea_datasets/xquad/xquad.py --subset_id xquad.th
```

Reply:
That test isn't applicable for multilingual dataset, you may abort it and re-test on the aforementioned bash commands.

See this [Reviewing SOP docs](https://docs.google.com/document/d/1uVtus5MXeM2lf4MmlSGhSXIHZXVLCqoaK6nh6sMyC3w/edit) on the `Dataloader Reviewer SOP` section.

Reply:
Ahhh, I see. I didn't know that. I was following [this guide](https://github.com/SEACrowd/seacrowd-datahub/blame/3eea2e750b867d4176624ebd3e2976c5e1384040/CONTRIBUTING.md#L165). We might wanna update that line.